system/nxlooper/nxlooper.c

```diff
@@ -24,21 +24,21 @@
 
 #include <nuttx/config.h>
 
-#include <sys/param.h>
-#include <sys/types.h>
-#include <sys/ioctl.h>
-
-#include <stdint.h>
+#include <assert.h>
+#include <debug.h>
+#include <dirent.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <sched.h>
 #include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
```

Comment:
I think that strings.h includes string.h, so there is no need to include both

Reply:
Even though NuttX coding standard imposes no restrictions or rules on this, I usually like to follow Google's C Style guide, in this case specifically the [Include What You Use guideline](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use).
It is better to be pedantic and include everything that is being referenced than to rely on transitive inclusions.

Reply:
Ok.